js/log_filters.js

```diff
@@ -56,7 +56,13 @@ $(function() {
             }
         });
 
-        $('.log_history_filter_row [name^="filters\\["]').on('input', handleFilterChange);
+        var delay_timer = null;
+        $('.log_history_filter_row [name^="filters\\["]').on('input', function() {
+            clearTimeout(delay_timer);
+            delay_timer = setTimeout(function() {
+                handleFilterChange();
+            }, 800);
```

Comment:
800ms seems a bit long to me, users may think it is lagging.
```suggestion
            }, 500);
```

Reply:
i tested originally with 500ms, it seemed a little too short